src/app/components/Loader/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Spinner from 'react-spinkit';
+
+import styles from './styles.module.scss';
+
+export default function Loader() {
+  return (
+    <div className={styles['loader-container']}>
```

Comment:
For compound class names, in your scss files name it with kebab-case like you do now and in your js files call them with camelCase.

```suggestion
    <div className={styles.loaderContainer}>
```